docs/developer-guide/coal-setup.md

```diff
@@ -48,24 +89,46 @@ At a high level the CoaL setup procedure is:
         ```
 
 
-1. Start VMware and load the appliance.
+1. Set memory and run.
 
-    - Mac: 'open'ing the folder will start VMware and "open and run" the vm:
+    - Mac example:
 
-            open coal-<branch>-<build_date_time>-<git_sha1_hash>-4gb.vmwarevm
+		1. Launch VMware Fusion
+		2. File > Open... `coal-<branch>-<build_date_time>-<git_sha1_hash>-4gb.vmwarevm`
+        3. Virtual Machine > Settings
+        4. Processes & Memory > set memory to a minimum 8192 MB.
 
-1. When you are prompted with the GRUB menu press the "down" arrow.
+1. When you are prompted with the GRUB menu press the down arrow.
 
-    1. Select the "Live 64-bit" option and press 'c' to enter the command
-       line for GRUB. By default, the OS will be redirect the console to
-       be ttyb which is fine for production but needs to be changed for
-       CoaL. While in the command line:
+  1. Select the "Live 64-bit" option and press 'c' to enter the command
+     line for GRUB.
 
-            grub> variable os_console vga
+     ![CoaL Grub Boot Menu](../img/coal-grub-menu.png)
 
-    1. Press ESC to get back to the GRUB menu.
+     By default, the OS will redirect the console to ttyb which is fine
+     for production but needs to be changed for CoaL. At the command line enter
+     "variable os_console vga":
+
```

Comment:
Since this is the larger guide, any reason to not also talk about using sercons here or in an appendix? At the cost of increasing steps, it will provide a better experience. Such as being able to copy and paste.